anxcloud/struct_dns_records.go

```diff
@@ -0,0 +1,27 @@
+package anxcloud
+
+import "github.com/anexia-it/go-anxcloud/pkg/clouddns/zone"
+
+func flattenDnsRecords(records []zone.Record) []interface{} {
+	zoneRecords := []interface{}{}
+	if len(records) < 1 {
+		return zoneRecords
+	}
+
+	for _, record := range records {
+		m := map[string]interface{}{}
```

Comment:
with the changes in the comment above, consider writing the always-present attributes like this:

```go
m := dnsRecordData{
  "identifier": ...,
  "name": ...,
}
```
only adding the TTL as you do now. IMHO this looks nicer, but it's perfectly valid to not agree :)